src/Features/CSharp/Portable/Completion/CompletionProviders/SpeculativeTCompletionProvider.cs

```diff
@@ -98,6 +106,14 @@ private async Task<bool> ShouldShowSpeculativeTCompletionItemAsync(Document docu
             }
 
             return false;
+
+            static void MoveTestPositionIfRefType(SyntaxToken tokenLeftOfType, ref int testPosition)
```

Comment:
be `int` returning instead of taking ref.

Reply:
No problem.